sklego/preprocessing/identitytransformer.py

```diff
@@ -0,0 +1,31 @@
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.preprocessing import FunctionTransformer
+
+from sklearn.utils import check_array
+from sklearn.utils.validation import FLOAT_DTYPES, check_is_fitted
+
+
+def _identity(X):
+    return X
+
+
+class IdentityTransformer(BaseEstimator, TransformerMixin):
+    """
+    The identity transformer returns what it is fed. Does not apply anything useful.
+    The reason for having it is because you can build more expressive pipelines.
+    """
+
+    def fit(self, X, y=None):
+        """'Fits' the estimator."""
+        X = check_array(X, copy=True, estimator=self)
+        self.fitted_ = True
```

Comment:
`fitted_` isn't needed if you also check `shape_` right?

Reply:
fixing now